app/change_set_persisters/plum_change_set_persister.rb

```diff
@@ -17,14 +17,19 @@ def self.registered_handlers
         PropagateVisibilityAndState
       ],
       after_save: [
-        AppendToParent
+        AppendToParent,
+        PublishMessage::Factory.new(operation: :update)
       ],
       before_delete: [
         CleanupMembership::Factory.new(property: :member_of_collection_ids),
         CleanupMembership::Factory.new(property: :member_ids)
       ],
+      after_delete: [
+        PublishMessage::Factory.new(operation: :delete)
+      ],
       after_commit: [
-        Characterize
+        Characterize,
+        PublishMessage::Factory.new(operation: :create)
```

Comment:
So I think we need things like `after_save_commit` and `after_delete_commit`. We don't want to fire messages for objects that are still locked up in a transaction.